Documentation/QebHwtThamosAdviseInput.md

```diff
@@ -0,0 +1,12 @@
+# QebHwtThamosAdviseInput
+
+## Properties
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**check_run_id** | **int** | ID of the check-run that shall carry the thamos advise. | 
+**repo_url** | **str** | URL of the GitHub repository containing the Pull Request. | 
+**commit_sha** | **str** | sha of the Pull Request&#x27;s head commit. | 
+**installation_id** | **int** | GitHub App Installation ID. | 
```

Comment:
This is old, see: https://github.com/thoth-station/user-api/pull/759/files

Reply:
It needs to be merged in order to regeneratge.

Reply:
So do we want to merge first? :)

Reply:
I don't mind, its just one script call :) Let bot merge it.